tests/common/osutil/test_default.py

```diff
@@ -657,16 +658,19 @@ def set_command(command, output='', exit_code=0):
         wait = FlexibleVersion(version) >= osutil._IPTABLES_LOCKING_VERSION
         uid = 42
 
-        version_command = set_command(osutil._get_iptables_version_command(), output=str(version))
-        list_command = set_command(osutil._get_firewall_list_command(wait), output="Mock Output")
-        set_command(osutil._get_firewall_packets_command(wait))
-        set_command(osutil._get_firewall_drop_command(wait, "-C", destination))
-        set_command(osutil._get_firewall_drop_command(wait, "-A", destination))
-        set_command(osutil._get_firewall_accept_command(wait, "-A", destination, uid))
+        version_command = set_command(osutil.get_iptables_version_command(), output=str(version))
+        list_command = set_command(osutil.get_firewall_list_command(wait), output="Mock Output")
+        set_command(osutil.get_firewall_packets_command(wait))
+        set_command(osutil.get_firewall_drop_command(wait, AddFirewallRules.CHECK_COMMAND, destination))
+        set_command(osutil.get_firewall_drop_command(wait, AddFirewallRules.APPEND_COMMAND, destination))
+        set_command(osutil.get_firewall_accept_command(wait, AddFirewallRules.APPEND_COMMAND, destination, uid))
+        set_command(osutil.get_accept_tcp_rule("-w" if wait else "", AddFirewallRules.APPEND_COMMAND, destination))
+        set_command(osutil.get_accept_tcp_rule("-w" if wait else "", AddFirewallRules.INSERT_COMMAND, destination))
```

Comment:
Why dont you also add the `CHECK_COMMAND` here too? 

i.e. - 
> set_command(osutil.get_accept_tcp_rule("-w" if wait else "", AddFirewallRules.CHECK_COMMAND, destination))

Reply:
Addressed